Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -35,10 +35,11 @@ namespace gdjs {
     private _rightKey: boolean = false;
     private _upKey: boolean = false;
     private _downKey: boolean = false;
-    private _leftKeyPressedDuration: integer = -1;
-    private _rightKeyPressedDuration: integer = -1;
-    private _upKeyPressedDuration: integer = -1;
-    private _downKeyPressedDuration: integer = -1;
+    private _leftKeyPressedDuration: integer = 0;
+    private _rightKeyPressedDuration: integer = 0;
+    private _upKeyPressedDuration: integer = 0;
+    private _downKeyPressedDuration: integer = 0;
```

Comment:
```suggestion
    private _leftKeyPressedDuration: float = 0;
    private _rightKeyPressedDuration: float = 0;
    private _upKeyPressedDuration: float = 0;
    private _downKeyPressedDuration: float = 0;
```

Reply:
`getElapsedTime` gives milliseconds.

Reply:
Well I think we can still have sub milliseconds precision?